ogr/ogrgeojsonwriter.cpp

```diff
@@ -1774,8 +1777,21 @@ char *OGR_G_ExportToJson(OGRGeometryH hGeometry)
  * (added in GDAL 3.9)</li>
  * <li>Z_COORD_PRECISION=integer: number of decimal figures for Z coordinates
  * (added in GDAL 3.9)</li>
- * <li>SIGNIFICANT_FIGURES=number:
- * maximum number of significant figures (GDAL &gt;= 2.1).</li>
+ * <li>SIGNIFICANT_FIGURES=number: maximum number of significant figures.</li>
+ * <li>ALLOW_CURVE=YES/NO: whether curve geometries are allowed. When set to NO
+ * (its default value), they are converted to linear geometries first.
+ * (added in GDAL 3.12.1)</li>
+ * <li>ALLOW_MEASURE=YES/NO: whether the measure (M) component of geometries is
+ * allowed. When set to NO (its default value), it is dropped when present.
+ * (added in GDAL 3.12.1)</li>
+ * <li>COORDINATE_ORDER=GIS_FRIENDLY/AUTHORITY_COMPLIANT (added in GDAL 3.12.1):
```

Comment:
Stay consistent with the OSR API which uses `TRADITIONAL_GIS_ORDER` and `AUTHORITY_COMPLIANT` ?